azure-pipelines/signing.yml

```diff
@@ -473,6 +473,42 @@ jobs:
         PathtoPublish: '$(Build.ArtifactStagingDirectory)\drop'
         ArtifactName: 'Drop'
         publishLocation: 'Container'
+    - task: CopyFiles@2
+      displayName: 'Copy Packaging Solution to Staging Area'
+      inputs:
+        SourceFolder: '$(Build.SourcesDirectory)/azure-pipelines/vs-insertion'
+        TargetFolder: '$(Build.ArtifactStagingDirectory)/vs-insertion/staging'
+        Contents: 'vcpkg.nuspec'
+        CleanTargetFolder: true
+    - task: CopyFiles@2
+      displayName: 'Copy vcpkg drop to Staging Area'
+      inputs:
+        SourceFolder: '$(Build.ArtifactStagingDirectory)/drop'
+        TargetFolder: '$(Build.ArtifactStagingDirectory)/vs-insertion/staging'
+        Contents: |
+          vcpkg-standalone-bundle.tar.gz
+          vcpkg.exe

```

Comment:
Is this going to have the right architecture?

Reply:
No, I don't think we need it